modules/container-converter/assets/js/editor/commands/convert-all.js

```diff
@@ -0,0 +1,18 @@
+import CommandHistory from 'elementor-document/commands/base/command-history';
+
+export class ConvertAll extends CommandHistory {
+	getHistory() {
+		return {
+			type: __( 'Converted to Containers', 'elementor' ),
+			title: __( 'Page', 'elementor' ),
```

Comment:
Like on Delete All Content
```suggestion
			title: __( 'All Content', 'elementor' ),
```

Reply:
Done

Reply:
Please replace the title, not the type.

Reply:
Oops